### ISSUE TITLE
trouble compilation

### ISSUE THREAD

**Issue by texrg** [esseks/monicelli#31]
Hello, I use ubuntu and have trouble:

```
$ cmake ..
== Only a limited set of platforms was tested. We need your help!
== Report build errors to https://github.com/esseks/monicelli/issues
-- Boost version: 1.54.0
-- Found the following Boost libraries:
--   regex
--   system
--   filesystem
--   program_options
CMake Error at /usr/share/llvm-3.5/cmake/LLVMConfig.cmake:50 (include):
  include could not find load file:

    /usr/lib/llvm-3.5/share/llvm/cmake/LLVMExports.cmake
Call Stack (most recent call first):
  src/CMakeLists.txt:34 (find_package)


CMake Error at /usr/share/llvm-3.5/cmake/LLVMConfig.cmake:53 (include):
  include could not find load file:

    /usr/lib/llvm-3.5/share/llvm/cmake/LLVM-Config.cmake
Call Stack (most recent call first):
  src/CMakeLists.txt:34 (find_package)

CMake Error at src/CMakeLists.txt:74 (llvm_map_components_to_libnames):
  Unknown CMake command "llvm_map_components_to_libnames".

-- Configuring incomplete, errors occurred!
```

I have llvm-3.5 and the file exists:

```
$ls -l /usr/share/llvm-3.5/cmake/LLVMConfig.cmake
-rw-r--r-- 1 root root 13472 lut 12  2015 /usr/share/llvm-3.5/cmake/LLVMConfig.cmake
```

what is wrong?

**Comment by robertodormepoco** [comment]
the missing file is /usr/lib/llvm-3.5/share/llvm/cmake/LLVMExports.cmake not /usr/share/llvm-3.5/cmake/LLVMConfig.cmake

**Comment by esseks** [comment]
This is a bug in the Ubuntu/Debian packaging process with two open tickets assigned, as [documented in the README](https://github.com/esseks/monicelli#building-with-llvm-on-debianubuntu). Until they fix this, llvm-3.5 from the LLVM APT repo works just fine.

Sorry for the incovenience, unfortunately there is nothing I can do.

**Comment by esseks** [comment]
The newest release uses a version of LLVM which shouldn't be affected by the bug. I'm closing this bug as obsolete, but feel free to reopen if it reoccurs.
